src/Microsoft.Dnx.Host.CoreClr/DomainManager.cs

```diff
@@ -11,8 +12,24 @@
 [SecurityCritical]
 sealed class DomainManager
 {
+    [StructLayout(LayoutKind.Sequential)]
+    private unsafe struct NativeBootstrapperContext
+    {
+        public char* OperatingSystem;
```

Comment:
As useful as this data is, and we should keep it, I think I'd prefer to have Runtime ID be a first-class member of this struct. Rather than just providing the components (OS Name, Ver, Arch, etc.), I'd like a field for the `RuntimeIdentifier` itself.